### DISCUSSION TITLE
Configure fzf with config file instead of env variables

### DISCUSSION THREAD

**Discussion by fritzrehde** [junegunn/fzf#3009] [Q&A]
Is there a way to configure fzf with a config file instead of environment variables? I want to implement a light dark mode toggle, and therefore need to change the colors from a terminal in all future fzf instances, even those not started from that terminal. However, any env vars set in that terminal would only configure fzf launched in that terminal. This is where a global config file would be useful.

**Comment by junegunn** [comment]
I think you can use a wrapper function or script named fzf for that purpose.

```sh
fzf() {
  # Load a file containing something like: `export FZF_DEFAULT_OPTS=...`
  source ~/.fzfrc 2> /dev/null

  # Dynamically applying different options
  if night; then
    command fzf --color dark "$@"
  else
    command fzf --color light "$@"
  fi
}
```

**Comment by eMPee584** [comment]
( This was implemented in https://github.com/junegunn/fzf/pull/3618 )
